org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
```

Comment:
Somehow the wrong line got highlighted by my comment. The **delay** package belongs in devDependencies. The **opent2t** package belongs in depenencies, because it can be used at runtime if the interface is loaded for reflection.

---
In reply to: [73765884](https://github.com/openT2T/translators/pull/37#discussion_r73765884) [](ancestors = 73765884)